addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -289,7 +289,13 @@ def _import_fattura_pa(self, tree, invoice):
                 partner = elements and self.env['res.partner'].search(['&', ('vat', 'ilike', elements[0].text), '|', ('company_id', '=', company.id), ('company_id', '=', False)], limit=1)
                 if not partner:
                     elements = tree.xpath('//CedentePrestatore//CodiceFiscale')
-                    partner = elements and self.env['res.partner'].search(['&', ('l10n_it_codice_fiscale', '=', elements[0].text), '|', ('company_id', '=', company.id), ('company_id', '=', False)], limit=1)
+                    codice = elements[0].text
+                    normalized_codice = self.env['res.partner']._l10n_it_normalize_codice_fiscale(codice)
+                    partner = elements and self.env['res.partner'].search([
+                        '&',
+                        '|', ('l10n_it_codice_fiscale', '=', codice), ('l10n_it_codice_fiscale', '=', normalized_codice),
```

Comment:
codice should be replaced by  "IT" + normalized_codice
```suggestion
                        '|', ('l10n_it_codice_fiscale', '=', "IT" + normalized_codice), ('l10n_it_codice_fiscale', '=', normalized_codice),
```

Reply:
Ok, I see what you mean. I tried to do something simple where we would only treat the case where the codice in the xml is not normalized.

What you suggest doesn't work, because if we have a physical person codice, this is weird. I tried to do something smarter instead.